MSON Specification.md

```diff
@@ -439,11 +439,14 @@ Defines extra attributes associated with the implementation of a type.
 - `required` - instance of this type is required
 - `optional` - instance of this type is optional (default)
 - `fixed`    - instance of this type structure and values are fixed
+- `nullable` - instance of this type *Value* MAY be unset (e.g. `null` or `nil`)
 - `sample`   - Alternate way to indicate a _[Value][]_ is a sample. See _[Sample][]_.
 - `default`  - Alternate way to indicate a _[Value][]_ is a default. See _[Default][]_.
 
 A `sample` _Type Attribute_ is mutually exclusive with `default`.
 
+Note by default, an instance of type SHOULD NOT be unset (`null`, `nil`).
```

Comment:
```
an instance of type SHOULD NOT be unset
```

The sentence doesn't sound legible. Did you mean value of an instance?

On second thought, do we really need to mention this at all? Isn't this given according to MSON spec?